lib/screens/authenticate/authentic.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:brew_crew/screens/authenticate/register.dart';
+import 'package:brew_crew/screens/authenticate/sign_in.dart';
+import 'package:flutter/material.dart';
+
+class Authenticate extends StatefulWidget {
+  @override
+  _AuthenticateState createState() => _AuthenticateState();
+}
+
+class _AuthenticateState extends State<Authenticate> {
+
+  bool showSignIn = false;
+
+  void toggleView(){
```

Comment:
kto to wywołuje? jeśli chcesz dać  widgetom "pod" tobą możliwość komunikacji z tobą, podaj do nich callbacka `onSignedIn`, `onLoggedOut`